pages/axes.py

```diff
@@ -5,21 +5,23 @@
 
 dash.register_page(
     __name__,
-    description="Interactive Plotly Axes Labels",
+    description="Interactive Plotly Axes Labels example in a bar plot. This app has a top-bottom layout and a regular-callback.",
 )
 
 filename = __name__.split("pages.")[1]
 
 
 notes = """
 
-#### Plotly Documentation:  
+#### Dash Components in App:  
+- [Dcc.Dropdown](https://dash.plotly.com/dash-core-components/dropdown)
```

Comment:
I can't see a dropdown in the code. But I saw a button. Can you please replace this link with the link to the html.button. https://dash.plotly.com/dash-html-components/button